core/cmd/starknet_keys_commands.go

```diff
@@ -13,7 +13,7 @@ type StarkNetKeyPresenter struct {
 
 // RenderTable implements TableRenderer
 func (p StarkNetKeyPresenter) RenderTable(rt RendererTable) error {
-	headers := []string{"ID", "Public key"}
+	headers := []string{"ID", "Public key", "Stark key"}
```

Comment:
We need better descriptions here. This data points are the public key and the derived account address from the public key, right?

If yes, let's describe them as such. `"Stark key"` is confusing IMHO.

Reply:
renamed it to be more explicit:
`Contract Address` and `Stark Key`